### PR TITLE
show an error message if a case is missing its creating form

### PR DESCRIPTION
**Pull request by czue**
http://manage.dimagi.com/default.asp?171041

image

### CODE REVIEW

File: corehq/apps/reports/views.py [hunk 1]
```diff
@@ -910,6 +911,17 @@ def case_details(request, domain, case_id):
         messages.info(request, "Sorry, we couldn't find that case. If you think this is a mistake please report an issue.")
         return HttpResponseRedirect(CaseListReport.get_url(domain=domain))
 
+
+    create_actions = filter(lambda a: a.action_type == CASE_ACTION_CREATE, case.actions)
```
Comment (dimagimon) [inline corehq/apps/reports/views.py]: E303 too many blank lines (2)

File: corehq/apps/reports/views.py [hunk 2]
```diff
@@ -910,6 +911,17 @@ def case_details(request, domain, case_id):
...
+    if not create_actions:
+        messages.error(
+            request,
+            _("This case is missing its creating form. "
+              "This can happen if the creating form is archived but there are other forms that updated the case. "
+              "To fix this you can archive the other forms listed here."
+            )
```
Comment (dimagimon) [inline corehq/apps/reports/views.py]: E124 closing bracket does not match visual indentation

**Comment by czue** [comment]
blocking on final text

**Comment by czue** [comment]
ok, good to go

**Comment by czue** [comment]
@benrudolph :white_check_mark:

**Pull request merged**